src/Http/Controllers/AuthorizationController.php

```diff
@@ -45,12 +45,12 @@ class AuthorizationController
      * Create a new controller instance.
      *
      * @param  \League\OAuth2\Server\AuthorizationServer  $server
-     * @param  \Illuminate\Contracts\Routing\ResponseFactory  $response
+     * @param  \Laravel\Passport\Contracts\AuthorizationViewResponse  $response
      * @param  \Illuminate\Contracts\Auth\StatefulGuard  $guard
      * @return void
      */
     public function __construct(AuthorizationServer $server,
-                                ResponseFactory $response,
+                                AuthorizationViewResponse $response,
```

Comment:
You're tightening the contract here which is a breaking change.

Reply:
Would you like me to change to the master branch?

Reply:
@driesvints do we allow or document people to modify this controller?

Reply:
I guess not 👍